counter-app/src/main/java/camp/nextstep/edu/counter/CounterViewModel.kt

```diff
@@ -9,16 +9,19 @@ class CounterViewModel : ViewModel() {
 
     val count: LiveData<Int> get() = _count
 
+    private val _showToastMessage = MutableLiveData<Boolean>()
+    val showToastMessage: LiveData<Boolean>
+        get() = _showToastMessage
+
     fun upCount() {
         _count.value = _count.value?.plus(1)
     }
 
-    fun downCount(): Result {
+    fun downCount() {
         if (_count.value == 0) {
-            return Result(false, "count must be positive number")
+            _showToastMessage.value = true
```

Comment:
```suggestion
            _showToastMessage.value = Unit
```

Nit: _showToastMessage의 타입이 Boolean이지만, 실제로 Boolean이 유의미한 상태를 가지진 않아요.
이 코드를  처음보는 다른 사람은 Boolean의 어떤 의미가 있는지 고민하는 상황이 생길 수 있기 때문에 단순히 값의 변화만 생겼다는 Unit 타입을 수신하도록 변경해보는 것도 좋습니다.

Reply:
값에 변경을 알리기위해서 사용한 값이였는데 Unit으로 해도 되는군요! 감사합니당~